### PR TITLE
Crud api configurations

### PR DESCRIPTION
**Pull request by jonyang221**
@hay-0le @briansjkim @darjama 
- Could someone help take a look before I merge?  Thanks!

Extended existing API to support all CRUD operations.
Added POST, PUT, DELETE.  
Still need to write tests.

### CODE REVIEW

File: server/app.js [hunk 1]
```diff
@@ -7,25 +7,59 @@ const db = require('../db');
...
-
     res.status(200).send(results);
+  });
 
+});
+
+//handle put requests for specifc seller
+app.put('/api/seller/:sellerID', (req, res) => {
+
+  db.putSellerData(req.params.sellerID, (err, results) => {
```
Comment (briansjkim) [inline server/app.js]: I think you may have forgot to pass in 'values' that your putSellerData function takes in

File: server/app.js [hunk 2]
```diff
@@ -7,25 +7,59 @@ const db = require('../db');
...
+      console.log('error occured deleting seller info', err);
+      res.status(404).send('Error occured deleting seller info');
+    }
+    res.status(200).send(results);
+  });
+
+});
 app.listen(PORT, () => {
   console.log('Listening on port', PORT);
 });
```
Comment (briansjkim) [inline server/app.js]: Don't forget to add a new line at the end of each file for Github

File: server/app.js (hunk 1 above)
Comment (sungjinbkim) [inline server/app.js]: I think you may have forgot to pass in 'values' that your putSellerData function takes in

File: server/app.js (hunk 2 above)
Comment (sungjinbkim) [inline server/app.js]: Don't forget to add a new line at the end of each file for Github

**Pull request merged**